phd-tests/framework/src/test_vm/mod.rs

```diff
@@ -396,30 +440,43 @@ impl TestVm {
         &mut self,
         source: &Self,
         migration_id: Uuid,
-        timeout_duration: Duration,
+        timeout: MigrationTimeout,
```

Comment:
nit, take it or leave it: it might be nice to add an `impl From<Duration> for MigrationTimeout` and change this to 
```suggestion
        timeout: impl Into<MigrationTimeout>,
```
so that tests that want to set explicit timeouts can just call this with a `Duration`, rather than having to import `MigrationTimeout` and construct one.

not a big deal, just a tiny API quality-of-life improvement :)